### PR TITLE
[server] Add Support To Logout Remote Provider Session

### PR DESCRIPTION
**Pull request by theBeginner86**
Signed-off-by: Pranav Singh <pranavsingh02@hotmail.com>

This PR:
1. Updates Makefile env variable of remote provider URL: `localhost:9081` -> `127.0.0.1:9081` (considering Kratos sees them differently and to prevent any edge case for requests from `meshery server` --> `remote provider` specific to dev env)
2. Updates Login flow to persist `session_cookie` cookie returned by the remote provider
3. Updates ExtensionProxy function to add `session_cookie` in requests to the remote provider (for extension points like user_account)
4. Updates logout flow to send logout request (with `session_cookie` cookie in headers) to the remote provider to run logout flow

**Notes for Reviewers**

This PR fixes https://github.com/meshery/meshery/issues/6503

This change helps in working of **Account Deletion Capability**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by theBeginner86** [comment]
Demo:


https://user-images.githubusercontent.com/73700530/204111491-8394af7c-02a5-4c13-9455-36a01828223e.mov

**Comment by theBeginner86** [comment]
**Remaining Items:**

- [ ] Handle Errors more promptly
- [ ] Use better error messages. 
- [ ] Avoid Error status code 500

**Review by leecalcote — CHANGES_REQUESTED** [review]
This is a well-documented behavior update. Will you consider moving to generic terms / variable names given this update lands inside of an extension point?

File: server/models/remote_provider.go [hunk 1]
```diff
@@ -2941,24 +3018,52 @@ func (l *RemoteProvider) SMPTestConfigDelete(req *http.Request, testUUID string)
...
 	if err != nil {
 		return nil, err
 	}
+
+	// gets the ory_kratos_session cookie from request headers
+	// necessary to run flows in remote provider specific to user_account extension
+	oryKratosSession, err := req.Cookie("ory_kratos_session")
+
+	// if cookie is not found then gracefully skip adding it the new request headers
+	// considering not every endpoint in remote provider requires it
```
Comment (leecalcote) [inline server/models/remote_provider.go:3053]: I ❤️ these comments.

**Comment by leecalcote** [comment]
A related issue to note - https://github.com/meshery/meshery/pull/6623/checks?check_run_id=9862028249

**Review by leecalcote — APPROVED** [review]

**Pull request merged**